src/components/Sidebar.js

```diff
@@ -145,7 +149,8 @@ const Sidebar = (props) => {
       separator_after: o.separator_after,
     }))
 
-  let content = informationPageEdges
+  let content = courseInfoPageEdges
+    .concat(informationPageEdges)
```

Comment:
Information_pages should be above the new pages in the navigation.

Would remove the courseInfoPageEdges and just set a `sidebar_priority` for the new page and filter these (o.course_info_page) into the informationPageEdges on line 136.

Reply:
Implemented in bf35079. I originally inteded the all exercises page as a tool for course assistants, so it's designed to be a hidden page and a bit more technical. Right now in development mode it shows up between "Arvostelu and kokeet" and "Tukea tehtäviin" with no sidebar_priority set.
If the page should be visible in production, the seperator of "Ohjelmoinnin MOOC" should probably be moved from "Yleiset virheilmoitukset (`virheilmoitukset.md`) to the all exercises page and sidebar priority set to reflect that.

Reply:
b48ed9f includes some better styling, so it would not look too bad for a student-facing page.
Right now it's still marked as hidden, but if there is a suitable location for it in the sidebar I wouldn't oppose it being there.

Reply:
I would ask course lecturer if it's suitable there for the students as well and where to put it.